sirepo/package_data/static/js/radia.js

```diff
@@ -1061,11 +1061,14 @@ SIREPO.app.controller('RadiaVisualizationController', function (appState, errorS
         return (appState.models.simulation || {}).enableKickMaps === '1';
     };
 
+    self.isSolvable = function() {
+        return (appState.models.geometry || {}).isSolvable == '1';
```

Comment:
I've noticed you use this construct `(appState.models.<model> || {}).<field>` berfore. Out of curiosity, what is the use case? Is it to handle models not being loaded yet? If so, I think `appState.isLoaded() && appState.models.geometry.isSolvable` is clearer.

Reply:
57f8898

I won't hunt down all the other cases of this, deferring to #1608